src/getexchange.c

```diff
@@ -161,9 +164,27 @@ int getexchange(void)
         }
         nodelay(stdscr, FALSE);
 
-
 	switch (x) {
 
+	case 17:	// CTRL-Q
+	    {
+		if (qtcdirection & 1) {
+		    qtc_main_panel(RECV);
+		}
+		if (qtcdirection == 2) {
+		    qtc_main_panel(SEND);
+		}
```

Comment:
Same effect as callinput. If qtcdirection = 3, then (qtcdirection == 2) will be FALSE.